src/language-usability.rst

```diff
@@ -19,5 +19,29 @@ API Usability
 Program Comprehension
 =====================
 
+Algorithm Animation
+-------------------
+
+**Algorithm animation** is a powerful approach for exploring a program's behavior. Equipped with different techniques, it has been used with success in teaching computer science courses, designing and analyzing algorithms, producing technical drawings, tuning performance, and documenting programs.
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "brown1991zeus"
+
+ This paper presents a system for algorithm animation, *Zeus*. To a programmer, *Zeus* can be viewed as a domain-independent framework for associating multiple clien-defined views with a set of client-defined events, generated by a client program called the algorithm. According to the author, constructing animations in *Zeus* appears to be easy and straightforward. The concept of Objects make it easy to reuse views, and to build sophisticated views by composing and subclassing other views.
+
+.. container:: bib-item
+
+  .. bibliography:: language-usability.bib
+    :filter: key == "stasko1993methodology"
+
+ An **application-specific** view illustrates the semantics of a program, its fundamental methodologies, and its inherent application domain. The paper discusses why **application-specific** views are necessary for parallel program debugging and several requirements and challenges for such a system. Also, an animation methodology called POLKA is developed to show **application-specific** views help programmers rapidly assess the programs' correctness.
```

Comment:
* What does "its fundamental methodologies" mean? What is an "inherent" application domain? Are these phrases from the paper? Consider rephrasing to be more clear about what an "application-specific view" is in the context of this paper.
* How does this relate to Zeus?
* Was this system empirically evaluated? If not, state so explicitly as above.